src/mapgen_v7.cpp

```diff
@@ -400,7 +407,7 @@ Biome *MapgenV7::getBiomeAtPoint(v3s16 p)
 	return bmgr->getBiome(heat, humidity, groundlevel);
 }
 
-//needs to be updated
+
```

Comment:
i need to ask you, what's this comment for? I can't see what needs updating.
Mountains don't include base terrain, the generation has been combined but these functions are correct as they are.